examples/nlp/md/multi_label_classification.md

```diff
@@ -6,7 +6,7 @@
 **Description:** Implementing a large-scale multi-label text classification model.
 
 
-<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team\keras-io\blob\master\examples\nlp/ipynb/multi_label_classification.ipynb)  <span class="k-dot">�</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team\keras-io\blob\master\examples\nlp/multi_label_classification.py)
+<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team\keras-io\blob\master\examples\nlp/ipynb/multi_label_classification.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team\keras-io\blob\master\examples\nlp/multi_label_classification.py)
```

Comment:
`â€¢` is what you get if you encode  `•` UTF-8 and decode it as latin-1 [ref](https://www.i18nqa.com/debug/utf8-debug.html)

Maybe GitHub is displaying latin-1 here because it auto detects it on the old version. 

This file *is* utf-8 encoded now, it's the display here that's wrong.

Reply:
Interesting, it shows `â€¢` in teh files view, but not the comments view.